### DISCUSSION TITLE
Setting up Container OS   Fails with [ERROR] in line 90: exit code 0

### DISCUSSION THREAD

**Discussion by Enn0** [tteck/Proxmox#3761] [Help Needed]
Trying to install pi-hole LXC from `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/pihole.sh)"` .
Every time I do, I encounter the same error:

✓ Updated LXC Template List
 ✓ LXC Container 100 was successfully created.
 ✓ Started LXC Container
 / Setting up Container OS   
`[ERROR] in line 90: exit code 0: while executing command locale_line=$(grep -v '^#' /etc/locale.gen | grep -E '^[a-zA-Z]' | awk '{print $1}' | head -n 1)`

I found a reference to this line here: https://github.com/tteck/Proxmox/blob/d07353534663f87186eceaf8c4a76ce9e886e0df/misc/install.func#L90 but am still clueless as to what exactly is going wrong, or how to fix it.

So hopefully someone here can point me in the right direction?

**Comment by Enn0** [comment]
I appaear to have the same issue installing ANY LCX through the helper scripts here. :/

From inside the LCX container:
```
root@pihole:~# 
root@pihole:~# echo $locale_line

root@pihole:~# locale_line=$(grep -v '^#' /etc/locale.gen | grep -E '^[a-zA-Z]' | awk '{print $1}' | head -n 1)
root@pihole:~# echo $locale_line

root@pihole:~# 
```

When I do this on the PVE host I get en_US.UTF-8. 

Manually downloading and installing pi-hole from inside the error'ed container...
```
(wget -O basic-install.sh https://install.pi-hole.net
sudo bash basic-install.sh)
```
... works just fine.

**Reply by tteck** [reply]
Are you using SSH?

**Comment by Enn0** [comment]
I ttried both through SSH session and GUI shell on PVE itself. Same result.
